politeiawww/paywall.go

```diff
@@ -212,10 +215,21 @@ func (b *backend) checkForProposalPayments(pool map[uuid.UUID]paywallPoolMember)
 			continue
 		}
 
-		// Removed paywall from the in-memory pool if it has been marked as paid.
+		// Removed paywall from the in-memory pool if it has
+		// been marked as paid.
 		if !b.userHasValidProposalPaywall(user) {
 			userIDsToRemove = append(userIDsToRemove, userID)
 			log.Tracef("  removing from polling, user just paid")
+		} else if tx != nil {
```

Comment:
This condition is never met. I couldn't figure it out why but I am pretty sure `b.verifyProposalPayment` either returns nil for tx or if isn't new, the payment has already been confirmed.

Reply:
This condition is met if the paywall has not been satisfied and a payment tx has not been sent yet.

Reply:
I said that backwards.  This condition is met if a transaction has been sent but does not have the minimum number of confirmations.